### DISCUSSION TITLE
Container component size prop vs maxW prop

### DISCUSSION THREAD

**Discussion by runoncedev** [chakra-ui/chakra-ui#4747] [Q&A]
What's the purpose of the `size` prop of the `Container` component?

The documentation suggests using the `maxW` prop to set the size of a `Container`.

**Comment by primos63** [comment]
`size` does nothing on the `Container` and the same goes for `colorScheme` and `variant`. There are no variants for `Container`. This probably needs to get cleaned up in the docs.
